CONTRIBUTING.md

```diff
@@ -21,14 +21,6 @@ For more details follow the [Build and Release documentation](docs/build_release
 
 To get started, refer to the documentation for [setting up a development environment](docs/development.md).
 
-### General contribution guidelines
```

Comment:
why was this removed?

Reply:
Moved to development.md, linked from here